### ISSUE TITLE
configuring existing projects

### ISSUE THREAD

**Issue by joprice** [fable-compiler/Fable.Package.SDK#8]
The readme mentions Fable.Promise and Elmish as examples of candidates for the `library` type, but when checking these, the Fable.Package.SDK seems to be missing, and I can't resolve source locally. Is the intention that pretty much every package missing the sdk dependency should have it added, or any reason not to add it to some? If so, I can do some prs to configure any repos I interact with like Fable.Promise, Elmish, Fable.React.Native, etc, since I find it very convenient especially as a newcomer to view the be able to view source for learning.

**Comment by MangelMaxime** [comment]
The mentioned repositories were mentioned to illustrate what a library or a binding is.

They were not mentioned as exemple of repositories that are actually using Fable.Package.SDK.

**Comment by joprice** [comment]
That makes sense. That was just the context how I came upon this. The question I had is the recommendation for if they should be configured when they are found to be missing sources, or if there's a reason to not configure them.

**Comment by MangelMaxime** [comment]
No specific reason to not configure them.

In an ideal world, we have used something like Fable.Package.SDK since the beginning. I hope for the adoption to be done over time, one of the first steps to do would be to update [Fable documentation](https://fable.io/docs/your-fable-project/author-a-fable-library.html) to use the same content as the README or redirect to this repository.

If someone wants to also send PR to existing Fable packages, this could help too but there more than 480 packages 
😅. At least, they are the ones that are discoverable by https://fable.io/packages/ but there are probably more than that.

I am closing this issue because this is not something that is actionnable from this repository. Feel free to continue the discussion or re-open if you feel like I closed it too soon.
